keyboards/mechwild/puckbuddy/readme.md

```diff
@@ -0,0 +1,19 @@
+# PuckBuddy
+
+![PuckBuddy](https://i.imgur.com/iSVAHJzh.png)
+
+A macropad with a Cirque Glidepoint Trackpad in the middle, powered by the STM32 Blackpill.
+
+* Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
+* Hardware Supported: PuckBuddy v1.0
+* Hardware Availability: [PuckBuddy on MechWild](https://mechwild.com/product/puckbuddy/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```